package.json

```diff
@@ -42,5 +47,11 @@
 		"id": "7328a705-91fc-49e6-8293-da6f112e482d",
 		"publisherDisplayName": "GitHub",
 		"publisherId": "7c1c19cd-78eb-4dfb-8999-99caf7679002"
+	},
+	"devDependencies": {
+		"@primer/primitives": "^2.0.1"
+	},
+	"scripts": {
+		"build": "node src/index.js"
```

Comment:
The script to auto-generate the two themes. This can be hooked into a `prepublish` step.